### DISCUSSION TITLE
Bug report link for UI leads to the archived repository

### DISCUSSION THREAD

**Discussion by Viiprogrammer** [verdaccio/verdaccio#3735] [General]
![image](https://user-images.githubusercontent.com/17622604/232256558-d08aea1d-15a1-4ad8-9504-17392762da6a.png)
![image](https://user-images.githubusercontent.com/17622604/232256635-f32563c9-0f32-46ca-89df-36171b14aa4f.png)

**Comment by juanpicado** [comment]
Thanks for let me know I will fix it
